lookout/style/format/tests/test_features.py

```diff
@@ -25,6 +25,15 @@ def setUpClass(cls):
             cls.uast = bblfsh.Node.FromString(fin.read())
         cls.extractor = FeatureExtractor("javascript", parents_depth=2, siblings_window=5)
 
+    def test_parse_file2(self):
```

Comment:
Please rename to something more informative like test_parse_file_on_comments if comments are causing a problem for example.

Reply:
mmm, it is not only for comments. It is for random stuff. I rename it to `test_parse_file_exact_match`